package.json

```diff
@@ -36,14 +41,17 @@
         "build": "ts-node ./dev/scripts/build.ts",
         "buildRepo": "pnpm build && pnpm docgen && pnpm buildSite",
         "buildSite": "cd dev/arktype.io && pnpm install && pnpm build",
+        "buildAttest": "cd dev/attest && pnpm i && pnpm build",
         "docgen": "ts-node ./dev/scripts/docgen.ts",
-        "test": "ts-node ./dev/attest/cli.ts --coverage --cmd mocha",
-        "tnt": "ts-node ./dev/attest/cli.ts --coverage --skipTypes --cmd mocha",
-        "testRepo": " pnpm testBuild && pnpm testTsVersions && pnpm testAttest",
+        "test": "pnpm c8Config ts-node ./dev/attest/src/cli.ts --runner mocha",
```

Comment:
Does this work? Never seen something like this. We should just invoke it from a script or something though, pretty convoluted.

Should be able to use something more like what we had previously in attest CLI but we can just add it to our own test runner file in dev/scripts